Jenkinsfile

```
@@ -49,6 +52,7 @@ pipeline {
       steps {
         sh '''
           docker build --pull -t ${IMAGE_NAME}:${TAG_NAME} -t ${IMAGE_NAME}:latest .
+          docker build --pull -t ${GL_IMAGE_NAME}:${TAG_NAME} -t ${GL_IMAGE_NAME}:latest .
```

I think you've forgot the `-f Dockerfile.gitlab` here

```suggestion
          docker build --pull -t ${GL_IMAGE_NAME}:${TAG_NAME} -t ${GL_IMAGE_NAME}:latest -f Dockerfile.gitlab .
```

---

Done!